postgres_lambda_strategy/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "aws_region" {
```

Comment:
For the sake of example simplicity, we are trying to reduce the number of variables used, as these examples are not intended to be drag and drop solutions to be copied, but as reference material that can be understood at a glance

Let's simply use `us-east-1` in our examples